homework/Vasili_Zasinets/AbstractMath/src/main/java/com/tutrit/java/quickstart/abstractmath/MathOperationRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.tutrit.java.quickstart.abstractmath;
+
+import java.util.List;
+
+public class MathOperationRepository {
+
+    public static final List<MathOperation> mathOperationList = List.of(
```

Comment:
According Java Code Convencion if fields has group of modifiers "public static final" then field are writing as UPPER CASE